### PR TITLE
promote committers

### PR DESCRIPTION
**Pull request by dcalvin**
In accordance with our [Governance](https://github.com/chaos-mesh/chaos-mesh/blob/master/GOVERNANCE.md#committer), I am pleased to promote the following Committers for Chaos Mesh. I would like to hereby thank them for their great contributions to the project and look forward to seeing more of their growth and engagements. 

 # [g1eny0ung](https://github.com/g1eny0ung) (Glen Yang)

**What does becoming a Chaos Mesh Committer mean to you?**

Thanks to all of my people who work with me at Chaos Mesh, and I'm glad to be a committer. For me, being a committer allows me to be deeply involved in the open-source itself and what I do can help anyone in need. It makes me feel that I am doing something meaningful. I will continue to focus on the ease of use of Chaos Mesh in the future, and strive to make it better.

**Contribution Highlights**

I mainly developed Chaos Dashboard in Chaos Mesh, The following are some of the highlights:

1. My first commit: [#445](https://github.com/chaos-mesh/chaos-mesh/pull/445).

2. Continuous improvement of the dashboard: [#577](https://github.com/chaos-mesh/chaos-mesh/pull/577), [#593](https://github.com/chaos-mesh/chaos-mesh/pull/593), [#628](https://github.com/chaos-mesh/chaos-mesh/pull/628), [#898](https://github.com/chaos-mesh/chaos-mesh/pull/898), [#1029](https://github.com/chaos-mesh/chaos-mesh/pull/1029), [#1285](https://github.com/chaos-mesh/chaos-mesh/pull/1285).

3. What I am working on: [#1283](https://github.com/chaos-mesh/chaos-mesh/issues/1283). This is a To-Do list, which covers the changes that have been completed or need to be made to the dashboard in the future.

4. Design and maintain the official website of Chaos Mesh: [https://chaos-mesh.org/](https://chaos-mesh.org/)

# [Andrewmatilde](https://github.com/Andrewmatilde) (Shenan Zhang)

**What does becoming a Chaos Mesh Committer mean to you?**

It means my contribution to the chaos-mesh has been recognized.

**Contribution Highlights**

Works in HTTP-Chaos, specifically, I helped add the CRD and API of HTTP-Chaos. #871

# [ethercflow](https://github.com/ethercflow) (Wenbo Zhang)

**What does becoming a Chaos Mesh Committer mean to you?**

I feel honored and lucky to be involved in such an interesting and meaningful project early on.

**Please list the highlights of your work (e.g. notable PRs, involvement in the community, etc.) at Chaos Mesh, preferably with reference links.**

Support kernel chaos (https://github.com/chaos-mesh/chaos-mesh/pull/273), in which I contributed to adding kernel chaos, a new chaos type, to Chaos Mesh. 

# [WangXiangUSTC](https://github.com/WangXiangUSTC) (Xiang Wang)

**What does becoming a Chaos Mesh Committer mean to you?**

It means I have a deeper understanding of Chaos Mesh and Chaos Engineering, For example, how to develop a new Chaos feature and which point should we focus on.

And I can contribute more to the project, not only the development but also the community. I will promote the project and help new contributors get involved in the project.

**Contribution Highlights**

Implement Chaos Mesh Action: https://github.com/chaos-mesh/chaos-mesh-action 

Implement DNSChaos: https://github.com/chaos-mesh/chaos-mesh/pull/1039

dashboard: support token: https://github.com/chaos-mesh/chaos-mesh/pull/1188 

speed up the e2e test: https://github.com/chaos-mesh/chaos-mesh/pull/1279 

make e2e test more stable: https://github.com/chaos-mesh/chaos-mesh/pull/1355

# [fewdan](https://github.com/fewdan) (Yihao Fu)

**What does becoming a Chaos Mesh Committer mean to you?**

It is an honor for me to become a Chaos Mesh Committer, which means that the community recognizes me. I will do my best to make more contributions to make Chaos Mesh better and better. It is a meaningful thing to be able to do a project better with the community.

**Contribution Highlights**

My main work is related to Chaos Mesh Dashboard, such as various APIs on the back end. At the same time, there are some specific chaos that I contributed to, such as container-kill and AwsChaos. Specifically, container-kill chaos expands PodChaos, allowing users to kill specific containers with more fine-grained chaos (PR #203). And AwsChaos helps users inject chaos into some components of AWS, such as EC2-stop and EC2-restart (PR #1330).

# [STRRL](https://github.com/STRRL) (Zhiqiang Zhou)

**What does becoming a Chaos Mesh Committer mean to you?**

I'm very pleased to be the committer of Chaos Mesh. It means my contribution on Chaos Mesh wins the approval of maintainers.

**Contribution Highlights**

- supporting namespace scoped chaos https://github.com/chaos-mesh/chaos-mesh/pull/872

- chaos mesh workflow(WIP) https://github.com/chaos-mesh/chaos-mesh/pull/1418

# [Yiyiyimu](https://github.com/Yiyiyimu) (Shuyang Wu)

**What does becoming a Chaos Mesh Committer mean to you?**

It's such an honor to become a Chaos Mesh committer and I'm so glad that my work could help people. I'll try to contribute more to the community and look forward to it becoming better!!

**Contribution Highlights**

1. develop a debug tool `chaosctl debug` to help collect debug information (#1074)

2. fix(ed&ing) several two-phase controller related issues (#1016, #1286, #1341)

# [Colstuwjx](https://github.com/Colstuwjx) (Jiaxing Wu)

**What does becoming a Chaos Mesh Committer mean to you?**

I'm also one of the users of chaos mesh. After being a Chaos Mesh Committer, I will get more actively involved in the community, and I could give more feedback both from user and developer sight. It's also a good chance to learn the operation of the community, how to make an elegant design of a complex system, and the most important thing, to work together with the community developers!

**Contribution Highlights**

I've been contributed to Chaos Mesh by the following PRs:

*  One of the missing features for the chaos spec is the expression selector support, and I did it, see [PR #1277]([https://github.com/chaos-mesh/chaos-mesh/pull/1277](https://github.com/chaos-mesh/chaos-mesh/pull/1277));

*  The Chaos Mesh package structure was not pretty good, and it causes some cycle-reference issues, see [issue 1072#comment]([https://github.com/chaos-mesh/chaos-mesh/issues/1072#issuecomment-711608326](https://github.com/chaos-mesh/chaos-mesh/issues/1072#issuecomment-711608326)), and I've contributed to removing some cross package references within [PR #1175]([https://github.com/chaos-mesh/chaos-mesh/pull/1175](https://github.com/chaos-mesh/chaos-mesh/pull/1175));

* I also contributed some fix PRs, for example, I fixed a bug in the two-phase controller after the PR merged, it makes the retry operation more robust, see [PR #946]([https://github.com/chaos-mesh/chaos-mesh/pull/946](https://github.com/chaos-mesh/chaos-mesh/pull/946)). I also tweaked the logging level while the controller catches a known bug after the PR merged, chaos controller wouldn't print lots of panic log due to the known issues, see [PR #914]([https://github.com/chaos-mesh/chaos-mesh/pull/914](https://github.com/chaos-mesh/chaos-mesh/pull/914));

* the other doc and chore PRs.

Besides the contribution of PRs, I also get involved in the review process. For example, I reviewed one of the new coming 2.0 features of Chaos Mesh, the Chaos Workflow, and I give some suggestions from design level and code level, see [PR 1418 #comment]([https://github.com/chaos-mesh/chaos-mesh/pull/1418#discussion_r559363810](https://github.com/chaos-mesh/chaos-mesh/pull/1418#discussion_r559363810)) and [rfc PR #10]([https://github.com/chaos-mesh/rfcs/pull/10](https://github.com/chaos-mesh/rfcs/pull/10)). I also helped review [PR #1221]([https://github.com/chaos-mesh/chaos-mesh/pull/1221](https://github.com/chaos-mesh/chaos-mesh/pull/1221)) and gave some useful comments.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Maintainers
+# Project Maintainers
```
Comment (zhouqiang-cl) [inline MAINTAINERS.md:1]: Should we put committers in MAINTAINERS.md?

Comment (cwen0) [reply MAINTAINERS.md:1]: I think it is ok. @dcalvin PTAL

File: MAINTAINERS.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
-This file lists the initial committers, and hence the maintainers of the Chaos Mesh project.
+This file lists the maintainers and committers of the Chaos Mesh project.
 
-In short, maintainers are people who are in charge of the maintenance of the Chaos Mesh project. This includes everything from day-to-day tasks such as issue triage to completing epics. Maintainers should be the first point of contact for security issues or other critical issues that require privacy not offered by the issue tracker.
+In short, maintainers are people who are in charge of the maintenance of the Chaos Mesh project.Committers are active community members who have shown that they are committed to the continuous development of the project through ongoing engagement with the community. For detailed description of the roles, see [Chaos Mesh Governance](https://github.com/chaos-mesh/chaos-mesh/blob/master/GOVERNANCE.md)
```
Comment (cwen0) [inline MAINTAINERS.md:5]: ...maintenance of the Chaos Mesh project. Committers       ...  

Add a space before `Committers`

**Review by cwen0 — COMMENTED** [review]
Rest LGTM

**Comment by dcalvin** [comment]
@Andrewmatilde Could you please comment and leave your org and email?

File: MAINTAINERS.md (hunk 1 above)
Comment (dcalvin) [reply MAINTAINERS.md:1]: I think it's good as these two are different roles with definitions.

**Comment by Andrewmatilde** [comment]
> @Andrewmatilde Could you please comment and leave your org and email?

OK,I leave it in my profile page.

File: MAINTAINERS.md [hunk 3]
```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
 Calvin Weng ([@dcalvin](https://github.com/dcalvin)) | [wenghao@pingcap.com](mailto:wenghao@pingcap.com) | [PingCAP](https://www.pingcap.com/)
 Ben Ye ([@yeya24](https://github.com/yeya24)) | [yb532204897@gmail.com](mailto:yb532204897@gmail.com) | Individual
 Hengliang Tan ([@Gallardot](https://github.com/Gallardot)) | [tttick@gmail.com](mailto:tttick@gmail.com) | [Xpeng Motors](https://www.xiaopeng.com/)
+
+
+The Committers of Chaos Mesh, along with their emails and focus area, are listed below:
+
+| Name                         | Email                    | Org     |
+|------------------------------|--------------------------|---------|
+| Glen Yang ([[G1eny0ung](https://github.com/g1eny0ung))  | yangyue@pingcap.com      | PingCAP |
```
Comment (Gallardot) [inline MAINTAINERS.md:26]:
```suggestion
| Glen Yang ([[G1eny0ung](https://github.com/g1eny0ung))  | yangyue@pingcap.com      | [PingCAP](https://www.pingcap.com/) |
```

Maybe we should keep the same style as above？

File: MAINTAINERS.md [hunk 4]
```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
...
 Hengliang Tan ([@Gallardot](https://github.com/Gallardot)) | [tttick@gmail.com](mailto:tttick@gmail.com) | [Xpeng Motors](https://www.xiaopeng.com/)
+
+
+The Committers of Chaos Mesh, along with their emails and focus area, are listed below:
+
+| Name                         | Email                    | Org     |
+|------------------------------|--------------------------|---------|
+| Glen Yang ([[G1eny0ung](https://github.com/g1eny0ung))  | [yangyue@pingcap.com](yangyue@pingcap.com)      | [PingCAP](https://www.pingcap.com/) |
+| Yihao Fu ([fewdan](https://github.com/fewdan))            | [fuyihao@pingcap.com](fuyihao@pingcap.com)      | [PingCAP](https://www.pingcap.com/) |
+| Jiaxing Wu ([Colstuwjx](https://github.com/Colstuwjx) )       | [wjx_colstu@hotmail.com](wjx_colstu@hotmail.com)             |   [Kingnet](www.kingnet.com)      |
```
Comment (Gallardot) [inline MAINTAINERS.md:28]:
```suggestion
| Jiaxing Wu ([Colstuwjx](https://github.com/Colstuwjx) )       | [wjx_colstu@hotmail.com](wjx_colstu@hotmail.com)             |   [Kingnet](https://www.kingnet.com/)      |
```

**Comment by cwen0** [comment]
@dcalvin Please fix [DCO](https://github.com/chaos-mesh/chaos-mesh/pull/1492/checks?check_run_id=1924973776). Thanks!

**Comment by dcalvin** [comment]
> @dcalvin Please fix [DCO](https://github.com/chaos-mesh/chaos-mesh/pull/1492/checks?check_run_id=1924973776). Thanks!

Done.

**Comment by dcalvin** [comment]
@Gallardot @cwen0 PTAL again.

File: MAINTAINERS.md [hunk 5]
```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
 Calvin Weng ([@dcalvin](https://github.com/dcalvin)) | [wenghao@pingcap.com](mailto:wenghao@pingcap.com) | [PingCAP](https://www.pingcap.com/)
 Ben Ye ([@yeya24](https://github.com/yeya24)) | [yb532204897@gmail.com](mailto:yb532204897@gmail.com) | Individual
 Hengliang Tan ([@Gallardot](https://github.com/Gallardot)) | [tttick@gmail.com](mailto:tttick@gmail.com) | [Xpeng Motors](https://www.xiaopeng.com/)
+
+
+The Committers of Chaos Mesh, along with their emails and focus area, are listed below:
+
+| Name                         | Email                    | Org     |
+|------------------------------|--------------------------|---------|
+| Glen Yang ([[G1eny0ung](https://github.com/g1eny0ung))  | [yangyue@pingcap.com](yangyue@pingcap.com)      | [PingCAP](https://www.pingcap.com/) |
```
Comment (Gallardot) [inline MAINTAINERS.md:26]:
```suggestion
| Glen Yang ([G1eny0ung](https://github.com/g1eny0ung))  | [yangyue@pingcap.com](mailto:yangyue@pingcap.com)      | [PingCAP](https://www.pingcap.com/) |
```

File: MAINTAINERS.md [hunk 6]
```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
...
+| Yihao Fu ([fewdan](https://github.com/fewdan))            | [fuyihao@pingcap.com](fuyihao@pingcap.com)      | [PingCAP](https://www.pingcap.com/) |
+| Jiaxing Wu ([Colstuwjx](https://github.com/Colstuwjx) )       | [wjx_colstu@hotmail.com](wjx_colstu@hotmail.com)             |   [Kingnet](www.kingnet.com/)      |
```
Comment (Gallardot) [inline MAINTAINERS.md:28]:
```suggestion
| Jiaxing Wu ([Colstuwjx](https://github.com/Colstuwjx))       | [wjx_colstu@hotmail.com](wjx_colstu@hotmail.com)             |   [Kingnet](https://www.kingnet.com/)      |
```

File: MAINTAINERS.md [hunk 7]
```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
...
+| Shuyang Wu ([Yiyiyimu](https://github.com/Yiyiyimu) )        | [wosoyoung@gmail.com]( wosoyoung@gmail.com)                        |  [APISIX](https://apisix.apache.org/)      |
+| Shenan Zhang ([Andrewmatilde](https://github.com/Andrewmatilde) | [zhangshenan@pingcap.com](zhangshenan@pingcap.com)   | [PingCAP](https://www.pingcap.com/)        |
```
Comment (Gallardot) [inline MAINTAINERS.md:30]:
```suggestion
| Shenan Zhang ([Andrewmatilde](https://github.com/Andrewmatilde)) | [zhangshenan@pingcap.com](zhangshenan@pingcap.com)   | [PingCAP](https://www.pingcap.com/)        |
```

File: MAINTAINERS.md [hunk 8]
```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
...
+| Xiang Wang ([WangXiangUSTC](https://github.com/WangXiangUSTC))   | [wangxiang@pingcap.com](wangxiang@pingcap.com)    | [PingCAP](https://www.pingcap.com/) |
+| Wenbo Zhang ([ethercflow](https://github.com/ethercflow) )     | [zhangwenbo@pingcap.com](zhangwenbo@pingcap.com)   | [PingCAP](https://www.pingcap.com/) |
```
Comment (Gallardot) [inline MAINTAINERS.md:32]:
```suggestion
| Wenbo Zhang ([ethercflow](https://github.com/ethercflow))     | [zhangwenbo@pingcap.com](zhangwenbo@pingcap.com)   | [PingCAP](https://www.pingcap.com/) |
```

File: MAINTAINERS.md [hunk 9]
```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
...
 Ben Ye ([@yeya24](https://github.com/yeya24)) | [yb532204897@gmail.com](mailto:yb532204897@gmail.com) | Individual
 Hengliang Tan ([@Gallardot](https://github.com/Gallardot)) | [tttick@gmail.com](mailto:tttick@gmail.com) | [Xpeng Motors](https://www.xiaopeng.com/)
+
+
+The Committers of Chaos Mesh, along with their emails and focus area, are listed below:
+
+| Name                         | Email                    | Org     |
+|------------------------------|--------------------------|---------|
+| Glen Yang ([[G1eny0ung](https://github.com/g1eny0ung))  | [yangyue@pingcap.com](yangyue@pingcap.com)      | [PingCAP](https://www.pingcap.com/) |
+| Yihao Fu ([fewdan](https://github.com/fewdan))            | [fuyihao@pingcap.com](fuyihao@pingcap.com)      | [PingCAP](https://www.pingcap.com/) |
```
Comment (Gallardot) [inline MAINTAINERS.md:27]:
```suggestion
| Yihao Fu ([fewdan](https://github.com/fewdan))            | [fuyihao@pingcap.com](mailto:fuyihao@pingcap.com)      | [PingCAP](https://www.pingcap.com/) |
```

Comment (Gallardot) [reply MAINTAINERS.md:27]: I think the rest also should be appended `mailto:`

**Review by Gallardot — CHANGES_REQUESTED** [review]
Rest LGTM

**Review by YangKeao — APPROVED** [review]
LGTM

**Review by fewdan — APPROVED** [review]
LGTM

File: MAINTAINERS.md [hunk 10]
```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
 Calvin Weng ([@dcalvin](https://github.com/dcalvin)) | [wenghao@pingcap.com](mailto:wenghao@pingcap.com) | [PingCAP](https://www.pingcap.com/)
 Ben Ye ([@yeya24](https://github.com/yeya24)) | [yb532204897@gmail.com](mailto:yb532204897@gmail.com) | Individual
 Hengliang Tan ([@Gallardot](https://github.com/Gallardot)) | [tttick@gmail.com](mailto:tttick@gmail.com) | [Xpeng Motors](https://www.xiaopeng.com/)
+
+
+The Committers of Chaos Mesh, along with their emails and focus area, are listed below:
+
+| Name                         | Email                    | Org     |
+|------------------------------|--------------------------|---------|
+| Glen Yang ([[G1eny0ung](https://github.com/g1eny0ung))  | [yangyue@pingcap.com](mailto:yangyue@pingcap.com)      | [PingCAP](https://www.pingcap.com/) |
```
Comment (fewdan) [inline MAINTAINERS.md:26]: There is an extra "[".  It should be "[G1eny0ung]" .

File: MAINTAINERS.md [hunk 11]
```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
...
+| Yihao Fu ([fewdan](https://github.com/fewdan))            | [fuyihao@pingcap.com](mailto:fuyihao@pingcap.com)      | [PingCAP](https://www.pingcap.com/) |
+| Jiaxing Wu ([Colstuwjx](https://github.com/Colstuwjx) )       | [wjx_colstu@hotmail.com](mailto:wjx_colstu@hotmail.com)             |   [Kingnet](https://www.kingnet.com/)      |
```
Comment (fewdan) [inline MAINTAINERS.md:28]: Should be "([Colstuwjx](https://github.com/Colstuwjx))" ?   It looks like there is an extra space.

File: MAINTAINERS.md [hunk 12]
```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
...
+| Shuyang Wu ([Yiyiyimu](https://github.com/Yiyiyimu) )        | [wosoyoung@gmail.com](mailto:wosoyoung@gmail.com)                        |  [APISIX](https://apisix.apache.org/)      |
```
Comment (fewdan) [inline MAINTAINERS.md:29]: ditto

File: MAINTAINERS.md [hunk 13]
```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
...
+| Shenan Zhang ([Andrewmatilde](https://github.com/Andrewmatilde) | [zhangshenan@pingcap.com](mailto:zhangshenan@pingcap.com)   | [PingCAP](https://www.pingcap.com/)        |
```
Comment (fewdan) [inline MAINTAINERS.md:30]: "Andrewmatilde" needs to add a ")" after it.  Should be "([Andrewmatilde](https://github.com/Andrewmatilde))"

File: MAINTAINERS.md [hunk 14]
```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
...
+| Xiang Wang ([WangXiangUSTC](https://github.com/WangXiangUSTC))   | [wangxiang@pingcap.com](mailto:wangxiang@pingcap.com)    | [PingCAP](https://www.pingcap.com/) |
+| Wenbo Zhang ([ethercflow](https://github.com/ethercflow) )     | [zhangwenbo@pingcap.com](mailto:zhangwenbo@pingcap.com)   | [PingCAP](https://www.pingcap.com/) |
```
Comment (fewdan) [inline MAINTAINERS.md:32]: Should be "([ethercflow](https://github.com/ethercflow))" ? It looks like there is an extra space.

**Review by fewdan — CHANGES_REQUESTED** [review]

**Comment by dcalvin** [comment]
@fewdan @Gallardot PTAL again. Thanks for being so perceptive.

**Review by Gallardot — APPROVED** [review]
LGTM

**Review by fewdan — APPROVED** [review]
LGTM

**Pull request merged**